tests/test_graph.py

```diff
@@ -213,3 +212,56 @@ def test_add_transform_invalid_type(graph: NeuromapsGraph, data_dir: Path) -> No
             key=graph.surface_to_surface_key,
             transform=surface_transform,  # type: ignore[arg-type]
         )
+
+@pytest.mark.usefixtures("require_data")
+def test_volume_atlases_exist(graph: NeuromapsGraph) -> None:
+    """Test that volume atlases exist and their paths are valid."""
+    for node_name in ["D99", "MEBRAINS", "NMT2Sym", "Yerkes19"]:
+        node = graph.get_node_data(node_name)
+        assert node.volumes, f"Node {node_name} should have volume atlases."
+
+        for atlas in node.volumes:
+            assert atlas.file_path.exists(), (
+                f"Volume atlas file {atlas.file_path} does not exist."
+            )
+
+@pytest.mark.usefixtures("require_data")
+def test_volume_to_volume_transform_objects_exist(graph: NeuromapsGraph) -> None:
+    """Test that volume-to-volume transforms exist and have valid paths."""
+    found = False
+
+    for _, _, key, edge_data in graph.edges(data=True, keys=True):
+        if key != graph.volume_to_volume_key:
+            continue
+
+        found = True
+        edge = edge_data["data"]
+        assert edge.volume_transforms, "Volume edge should contain transforms."
+
+        for transform in edge.volume_transforms:
+            assert transform.file_path.exists(), (
+                f"Volume transform file {transform.file_path} does not exist."
+            )
+
+    assert found, "Graph should contain volume_to_volume edges."
```

Comment:
Is this just to test if there are `volume_to_volume` edges? If so, can just assert the number of edges is greater than 1? Otherwise if testing each key contains an edge, should `found` not be reset? Just trying to understand this test.